internal/controller/apiextensions/composite/composed.go

```diff
@@ -133,6 +133,27 @@ func (r *APIDryRunRenderer) Render(ctx context.Context, cp resource.Composite, c
 	return errors.Wrap(r.client.Create(ctx, cd, client.DryRunAll), errName)
 }
 
+// An CompositeRenderer renders the composite resource
+type CompositeRenderer struct{} // nolint: golint
+
+// NewCompositeRenderer returns a Renderer composite resource
+func NewCompositeRenderer() *CompositeRenderer {
+	return &CompositeRenderer{}
+}
+
+// Render the supplied composite resource using the supplied composed resource
+// and template.
+func (r *CompositeRenderer) Render(_ context.Context, cp resource.Composite, cd resource.Composed, t v1.ComposedTemplate) error {
+	compositePatches := []v1.PatchType{v1.PatchTypeToCompositeFieldPath}
```

Comment:
Nit: I'd be tempted to call this `only` to clarify how it's used by `p.Apply` below.